src/CSharp/EasyMicroservices.ProfilesMicroservice.WebApi/Program.cs

```diff
@@ -1,168 +1,38 @@
-using EasyMicroservices.ProfilesMicroservice.Database;
 using EasyMicroservices.ProfilesMicroservice.Database.Contexts;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.Migrations;
-using Microsoft.OpenApi.Any;
-using Microsoft.OpenApi.Models;
-using Swashbuckle.AspNetCore.SwaggerGen;
-using System.Reflection;
-using Microsoft.Extensions.DependencyInjection;
-using EasyMicroservices.ProfilesMicroservice.Database.Entities;
-using EasyMicroservices.ProfilesMicroservice.Contracts;
-using EasyMicroservices.ProfilesMicroservice.Interfaces;
+using EasyMicroservices.Cores.AspEntityFrameworkCoreApi;
+using EasyMicroservices.Cores.Relational.EntityFrameworkCore.Intrerfaces;
 using EasyMicroservices.ProfilesMicroservice;
-using EasyMicroservices.ProfilesMicroservice.Contracts.Common;
-using EasyMicroservices.ProfilesMicroservice.Contracts.Responses;
-using EasyMicroservices.ProfilesMicroservice.Contracts.Requests;
 
-namespace EasyMicroservices.ProfilesMicroservice.WebApi
+namespace EasyMicroservices.ContentsMicroservice.WebApi
```

Comment:
it's Profile microservice, don't change the namespace please

Reply:
was a mistake